normandy/classifier/tests/test_api.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+
+from normandy.base.utils import Whatever
+from normandy.recipes.tests import RecipeActionFactory, RecipeFactory
+
+@pytest.mark.django_db
+class TestFetchBundleAPI(object):
+
+    def test_it_works(self, client):
```

Comment:
Yes. This is what py.test calls fixtures. It depends on the *name* of the parameter. If I had called this `foobar`, I would get something entirely different. Probably nothing. [Docs here](http://pytest-django.readthedocs.org/en/latest/helpers.html#fixtures)